### PR TITLE
HRINT-2918 [Demo] Spatial Demos (Java)

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

**Comment by Danielle9897** [comment]
@ml054 pls review

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/spatial/spatialIndex/SpatialIndex.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                this.companyName = companyName;
+            }
+        
+            public Object getLocationCoordinates () {
+                return locationCoordinates;
+            }
+        
+            public void setLocationCoordinates (Object locationCoordinates) {
+                this.locationCoordinates = locationCoordinates;
+            }
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/spatial/spatialIndex/SpatialIndex.java:36]: please reformat that code and auto generate again

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/spatial/spatialIndex/SpatialIndex.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+public class SpatialIndex {
+    //region Demo
+    //region Step_1
+    public static class Companies_ByLocation extends AbstractIndexCreationTask {
+    //endregion
+    
+        //region Step_2
+        public static class IndexEntry {
+            private String companyName;
+            private Object locationCoordinates ;
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/spatial/spatialIndex/SpatialIndex.java:20]: remove space before `;`

File: DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/spatialIndex/SpatialIndexTest.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import java.util.List;
+
+public class SpatialIndexTest {
+
+    @Test
+    public void test() {
+        DocumentStoreHolder.store.executeIndex(new SpatialIndex.Companies_ByLocation());
+
+        List<Company> companies = new SpatialIndex().run();
+        Assert.assertTrue(companies.size() == 4);
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/spatialIndex/SpatialIndexTest.java:18]: that's not the way we use assertions.

File: DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/spatialIndex/SpatialQueryTest.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+public class SpatialQueryTest {
+
+    @Test
+    public void test() {
+        SpatialQuery.RunParams runParams = new SpatialQuery.RunParams();
+
+        runParams.setRadius(2);
+
+        List<SpatialQuery.EmployeeDetails> results = new SpatialQuery().run(runParams);
+        Assert.assertTrue(results.size() == 2);
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/spatialIndex/SpatialQueryTest.java:18]: same here.

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/spatial/spatialIndex/SpatialIndex.java (hunk 2 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/spatial/spatialIndex/SpatialIndex.java:20]: done

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/spatial/spatialIndex/SpatialIndex.java (hunk 1 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/spatial/spatialIndex/SpatialIndex.java:36]: done

File: DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/spatialIndex/SpatialQueryTest.java (hunk 4 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/spatialIndex/SpatialQueryTest.java:18]: changed to `assertEquals`

File: DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/spatialIndex/SpatialIndexTest.java (hunk 3 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/test/java/net/ravendb/demo/test/spatialIndex/SpatialIndexTest.java:18]: changed to `assertEquals`

**Comment by Danielle9897** [comment]
For issue:
https://issues.hibernatingrhinos.com/issue/HRINT-2918/Demo-Spatial-Demos-Java

**Pull request merged**